migrations/0007-create-exhibitions.js

```diff
@@ -0,0 +1,240 @@
+module.exports = function(migration) {
+  const exhibitionPage = migration
+    .createContentType('exhibitionPage')
+    .name('Exhibition Page')
```

Comment:
Let's clarify the distinction with the chapter pages by naming this "exhibition landing page".